rgabeflores/rgabeflores.ino

```diff
@@ -16,7 +16,7 @@ void loop() {
     // The condition simulates a "sprinkler" motion by running the following
     //   instructions only when the desired position is divisible by 30
     //   (i.e pos = 30, pos = 60, pos = 90... etc.)
-    if( (pos % 30) == 0){
+    if(pos % 30 == 0){
```

Comment:
You are still missing some spaces around the parenthesis of the if statement.
```C++
if(pos % 30 == 0){
```
should be 
```C++
if (pos % 30 == 0) {
```

After updating that, you'll be golden! 👍